Pod/Classes/SideMenuManager.swift

```diff
@@ -42,6 +42,12 @@ open class SideMenuManager : NSObject {
      */
     open static var menuPresentMode: MenuPresentMode = .viewSlideOut
     
+    /// Retrieves each already created view controller to front when called. Requires class restoration identifiers. Defaults to false.
+    open static var menuPreserveViewOnPush = false
+    
+    /// Mimics a tab function, by instantly presenting the view controller. Defaults to false.
+    open static var menuTabMode = false
```

Comment:
The purpose of this seems to be enabling or disabling animations. This can already be done by turning off animations on the segue in storyboard, or by setting `animated: false` when pushing or presenting from a menu in code. As such, this appears to be redundant and therefor unnecessary.